theme/styles/snippets/cart-item.scss

```diff
@@ -36,4 +36,22 @@
       max-height: 17rem;
     }
   }
+
+  &__quantity-spinner, &__quantity-spinner button {
+    font-size: 2.5rem;
+    line-height: 2.625rem;
+
+    @include media('md-up') {
+      font-size: 1.5rem;
+      line-height: 1.8rem;
+      letter-spacing: 0.0275rem;
+    }
+  }
+
+  &__increment, &__decrement {
+    &[disabled] {
+      cursor: not-allowed;
+      pointer-events: all;
+    }
```

Comment:
Seems like you have to have `pointer-events: all` for the `cursor not-allowed` to appear!

Reply:
Ah gotcha, makes sense. Let's remove the cursor then, and maybe just add a slight opacity when disabled? Sorry about that! (We definitely want pointer-events: none)